1001-2000/LIT1295Dersan.xml

```diff
@@ -138,413 +146,1482 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                     </ab>
+                  </div>
                </div>
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamTahsas">
-                  <label xml:lang="en">
-                     <foreign xml:lang="gez">Salām</foreign> for <foreign xml:lang="gez">Tāḫśāś</foreign>
-                  </label>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamAqabe" corresp="LIT6822SalamLakaAqabe">
+                  <label xml:lang="en"><foreign xml:lang="gez">Salām</foreign>to you Saints</label>
+                  <ab>
+                     <l n="89">፹፱፤ ሰላም፡ ለከ፡ ዐቃቤ፡ ቅዱሳን፡ እምተሀውኮ፡ ወለፀረ፡ ነፍሶሙ፡ ዘሞቃሕኮ። አመ፡ መጽአ፡ ወልድ፡ በአፈ፡ ነቢያት፡ ተሰቢኮ፡ ከመ፡ ያስተፋኑ፡ ሐራ፡ መልአኮ፡
+                        እስከ ምጽንዓት፡ ኃምስ፡ ሚካኤል፡ ተለውኮ።</l>
+                  </ab>
                </div>
             </div>
             
             <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Terr">
-               <label xml:lang="en">
-                  <foreign xml:lang="gez">Ṭǝrr</foreign>
-               </label>
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyTerr">
+               <label xml:lang="en">Ṭǝrr</label>
+               <div type="textpart" subtype="part" xml:id="AdmonitionTerr">
+                  <label>Admonition for Ṭǝrr</label>
+                  <ab>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ ሥሉስ፡ ዘኢይትሌለይ፡ ዕሩይ፡ ዘኢይሴደቅ፡ ታሉት፡ ዘኢይት፡ በዐድ። ርድአኒ፡ ከመ፡ እንግር፡ ድርሳነ፡ ዘቅዱስ፡
+                     ወክቡር፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ወመልአከ፡ ኀይል፡ ዘርኡስ፡ ላዕለ፡ ኵሎሙ፡ ሠራዊተ፡ መላእክት፡ ሰማያውያን፡ ዘውእቶሙ፡ አእላፈ፡ አእላፋት፡ ዘይትነበብ፡ በበዓሉ፡ ለሊቀ፡
+                     መላእክት፡ ሚካኤል፡ አመ፡ ዐሡሩ፡ ወሰኑዩ፡ ለወርኀ፡ ጥር። ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ ዓለም፡ አሜን።</ab>
+               </div>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyTalafinos" corresp="LIT3959Homily">
                   <listRelation>
-                     <relation name="saws:isAttributedToAuthor" active="LIT1295Dersan#HomilyTerr" passive="PRS3828Epiphani"/>
+                     <relation name="saws:isAttributedToAuthor" active="LIT1295Dersan#HomilyTalafinos" passive="PRS3828Epiphani"/>
                   </listRelation>
-                  <label xml:lang="en">Homily for <foreign xml:lang="gez">Ṭǝrr</foreign> by <persName ref="PRS3828Epiphani"/>
-                     <!-- This homily is transmitted in the section for Ṭǝrr in BAVet82, but in the section for Miyāzyā in BAVet121  -->
-                  </label>
+                  <label xml:lang="en">Homily by Epiphanius</label>
+                  <note>Homily on the angels and the child <persName ref="PRS11321Talafinos"/>, who was found at the sea. 
+                     Anonymous in <bibl><ptr target="bm:2006DersanaMikael"/></bibl>, but elsewhere attributed to 
+                     <persName ref="PRS3828Epiphani"/>. 
+                        The story is a variant of <ref type="work" corresp="LIT4155MiraclesMichael"/>, 
```

Comment:
all this must (also) be expressed in relations, probably better from `LIT4155MiraclesMichael`